templates/user/dashboard/issues.tmpl

```diff
@@ -99,6 +99,9 @@
 								<a class="{{if eq .SortType "farduedate"}}active{{end}} item" href="{{$.Link}}?type={{$.ViewType}}&repos=[{{range $.RepoIDs}}{{.}}%2C{{end}}]&sort=farduedate&state={{$.State}}&q={{$.Keyword}}">{{.locale.Tr "repo.issues.filter_sort.farduedate"}}</a>
 							</div>
 						</div>
+						{{if .Repo}}
+							<a class="ui green button" href="{{AppSubUrl}}/{{.SingleRepoLink}}/issues/new">{{.locale.Tr "repo.issues.new"}}</a>
```

Comment:
```suggestion
							<a class="ui green button" href="{{.SingleRepoLink}}/issues/new/choose">{{.locale.Tr "repo.issues.new"}}</a>
```

Reply:
If there are issue templates, only the `/issues/new/choose` can open the templates page.